specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -477,6 +477,105 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers": {
```

Comment:
>customers [](start = 71, length = 9)

Whose customers are these? (No issue with naming - just trying to understand the relationships.)

Reply:
A (CSP) partner has a billing account. The customers resource lists the customers the partner does business with.

Reply:
Gotcha. In that case calling it "customer" definitely makes sense.

---
In reply to: [295015980](https://github.com/Azure/azure-rest-api-specs/pull/6299#discussion_r295015980) [](ancestors = 295015980)